docs/source/release/v3.7.0/reflectometry.rst

```diff
@@ -5,6 +5,11 @@ Reflectometry Changes
 .. contents:: Table of Contents
    :local:
 
+Reflectometry Instruments
+--------------------------
+
+An updated version of the OFFSPEC IDF is now being used in mantid `#15561 <https://github.com/mantidproject/mantid/pull/15561>_`
+   
```

Comment:
I noticed that it is a tradition to give the PR link in the reflectometry release notes, for all or most changes. I think we are not very consistent in this throughout the different areas. Other areas don't usually have links to issues/PRs, except in a few specific cases.